src/main/res/values/strings.xml

```diff
@@ -280,15 +280,15 @@
     <string name="comment_edit_done">Comment edited. Changes will appear after you refresh.</string>
     <string name="post_submit_title_hint">Title</string>
     <string name="post_submit_subreddit_hint">Subreddit</string>
-    <string name="post_captcha_title">Captcha</string>
+    <string name="post_captcha_title">CAPTCHA</string>
     <string name="post_captcha_submit_button">Submit</string>
     <string name="post_submit_done">Post submitted.</string>
 
     <!-- Bug Reporting -->
 
     <string name="bug_title">An error has occurred.</string>
-    <string name="bug_message">RedReader has encountered an error.\n\nWould you like to send a bug report to the developer? If so, an email will be filled in for you with all the relevant information, and you can inspect the contents before sending.</string>
-    <string name="bug_button_send">Send bug report by email</string>
+    <string name="bug_message">Would you like to send a bug report to the developer? If so, an e-mail with all the relevant info will be filled in for you, and you can inspect it all before sending.</string>
```

Comment:
Prefer the original wording

Reply:
It is already under "an error has occurred", so that part is redundant.
Then it self-references the program, which is bad practice. "contents" doesn't go beyond "relevant info", so that is superfluous. "Information" isn't different from "Info".
The sum of changes, which keep the wording the exact same for other parts, manages to be a lot shorter, which helps comprehension, and induces less stress.

Reply:
The sentence "((an email with X) will be filled in)" is semantically different to the original "(an email (will be filled in with X))". Also the second version uses the word "all" twice which sounds clunky.

Just seems like needless rewording of something that was marginally clearer before :p Let's just leave it as it was originally.

Reply:
True, but it breaks down to the same resulting e-mail, containing the same info.

Reply:
Let's keep it as it was, I don't think this is an improvement.

Reply:
@QuantumBadger It is 53 chars shorter, and _still_ verbose … ("Would you like to - a " can be omitted.) How is that not a whole lot better?

The original is awful. There is an "if so" conditional, hinging on the first sentence, and the second doesn't end before piling on "relevant" and "info", referencing that in "all"; which no questions answered …
The "you" is invoked for no good reason, and it is posed as a question that just creates doubt.
The whole thing pivots on a question that may or may not have something to be inspected, that the user should or should not already be knowledgeable about prior to answering.

**An error has occurred.**
Would you like to send a bug report to the developer? If so, an e-mail with all the relevant info will be filled in for you, and you can inspect it all before sending.
[Send bug report by email] [Don\'t send bug report]

**Reportable error**
Inspect the prefilled e-mail before sending it to the developer.
[Send] [Cancel]

The level of entropy here is "it" refers to both the e-mail and error. "Reportable" ties them together. There is a flow that doesn't treat the user as a dumpsite for thoughtless text.

There is an error, it is reportable. Sending it in happens via e-mail, and the e-mail needs to be inspected before it is sent. When it is sent, the recipient will be the developer.
Instead of the apologetic style, "reportable" is good. Fail graciously.

Reply:
@comradekingu We've spent way too long debating this. This appears for very rare, literally one-in-every-million-sessions errors, so verbosity is not an issue. Let's not spend any more time on it please!

Reply:
Great. I care and you don't, so that is you opting to put time into keeping it worse.
The original is worse for reasons like opting to use "bug report", which is not as common, and hard to translate in some languages. Keep it simple and solid and translators will put a lot more effort into trying to keep up with the quality.